src/lib/components/molecules/page-section/page-section.stories.jsx

```diff
@@ -0,0 +1,61 @@
+import { PageSection } from '.'
+import { Container } from '$particles'
+import styles from './stories.module.scss'
+
+const meta = {
+  title: 'Molecules/PageSection',
+  component: PageSection,
+  parameters: {
+    layout: 'fullscreen',
+    viewport: {
+      defaultViewport: 'reset',
+    },
+  },
+  render: (args) => {
```

Comment:
To make most use of Storybook - shall we make the header and content text editable? Eg could be useful for people testing out what headers would fit in this space. And fits more with how we have done the other components. 
Something like this? 
```
 args: {
    headerText: "Section one!!!",
    contentText: "Section context!!!!"
  },
  render: (args) => {
    return (
      <PageSection {...args}>
        {{
          header: <h2>{args.headerText}</h2>,
          content: <p className={styles.content}>{args.contentText}</p>,
        }}
      </PageSection>
    )
  },
  ```
  
<img width="577" alt="Screenshot 2024-03-05 at 12 39 05" src="https://github.com/guardian/interactive-component-library/assets/10324129/2e7f6f73-53a6-4337-8d5a-b5fe6de41e9e">

Reply:
I see your point, but that would go against what we discussed with Sean earlier, namely the composability of components. The idea that a components can function as "dumb" wrappers, without being aware of their contents. If we used regular text props instead of the children prop, then we would be baking that into the PageSection component, losing the composability. For instance, some section headers might need to contain a legend as well as a title, and in its current from the PageSection supports that, because it is content-agnostic (except for knowing of the distinction between header and content)

Reply:
Hmm - to my thinking this would only affect the presentation in Storybook - ie we're only editing the .stories file rather than the index.jsx. And would allow users to play around more. But it's not particularly important so happy to drop this suggestion 

(Tho wd still like to understand how editing only the .stories file would affect the component?)

Reply:
I'm not sure it's possible to completely decouple those controls from the actual component props, because that is what they are supposed to represent. Though I haven't tried, so it might be doable